resources/react/components/form/field-renderer.tsx

```
@@ -157,7 +157,7 @@ export const FieldRenderer = withForm({
           )
 
         case 'tel':
-          return <form.AppField name={schema.key} children={(field) => <field.TelField schema={schema} />} />
+          return <form.AppField name={schema.key} children={(field) => <field.TelField schema={schema} form={form} />} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)